app/services/support_interface/send_ucas_match_reminder_emails.rb

```diff
@@ -0,0 +1,40 @@
+module SupportInterface
+  class SendUCASMatchReminderEmails
+    attr_accessor :ucas_match
+
+    def initialize(ucas_match:)
```

Comment:
I think we can skip keyword arguments in this instance as the class name is pretty descriptive about being related to ucas_matches  (and to keep it consistent with the rest of the ucas_match related email services.